arguments_test.go

```diff
@@ -72,6 +75,15 @@ func TestJSONInputOutput(t *testing.T) {
 	assert.Equal(t, 20756, outObj.Value)
 }
 
+func TestReadNotEmpty(t *testing.T) {
+	// Note: The contents need to be larger than the default buffer size of bufio.NewReader.
```

Comment:
I don't really think it adds much value to have a constant that's only used in a single test, and doesn't represent anything (e.g., it's counting the number of repeats for the string, not a number of bytes). It's pretty arbitrary.